harper-core/src/linting/noun_verb_confusion/mod.rs

```diff
@@ -10,9 +10,16 @@ pub(crate) const NOUN_VERB_PAIRS: &[(&str, &str)] = &[
     ("advice", "advise"),
     ("belief", "believe"),
     ("breath", "breathe"),
+    ("device", "devise"),
     ("effect", "affect"), // "Effect" is also a verb meaning "to bring about". "Affect" is a noun in psychology.
     ("emphasis", "emphasize"), // TODO how to handle "emphasise" as well as "emphasize"?
+    ("envelop", "envelope"),
     ("intent", "intend"),
+    ("pour", "poor"),
+    ("pray", "prey"),
+    ("resume", "resumé"),
+    ("waste", "waist"),
+    ("write", "rite"),
```

Comment:
Except for "device" and "devise", these all have the verb first and the noun second, where it should be noun first and verb second. An easy way to remember is the name of the file as noun before verb.

"resume" as a noun has three spellings in English: `resume`, `résumé`, and `resumé`. Without further changes this module isn't designed to work with words that have multiple spellings. With my original "emphasis" vs "emphasize" I chose the most universal one and added a comment about the limitation. I don't think `resumé` is the most univerally accepted spelling in English but some research could prove otherwise.